lang/python/python3-dbus/patches/010-keep-config.patch

```diff
@@ -0,0 +1,47 @@
+--- a/setup.py
```

Comment:
Patch should be described like: 
1. why do we need it to have it in our repository
2. why are you doing this changes and why they are required
3. who is author

And why this can not be done in upstream.

Reply:
We no longer need to. As no one most likely does not need this and even I no longer need it, I will close this now from haunting here.. Though, check that python3-readline - it's still needed..